sqladmin/models.py

```diff
@@ -125,7 +126,24 @@ def is_accessible(self, request: Request) -> bool:
         return True
 
 
-class ModelAdmin(BaseModelAdmin, metaclass=ModelAdminMeta):
+class ModelView(BaseModelAdmin):
+    is_model = False
+
+    template_path: ClassVar[str]
+    templates: Jinja2Templates
+
+    name: ClassVar[str]
+    icon: ClassVar[str]
+    path: ClassVar[str]
+    methods = ["GET"]
+    endpoint: ClassVar[Callable]
+    include_in_schema: ClassVar[bool] = True
+
+    def render(self, template, data):
+        return self.templates.TemplateResponse(template, data)
```

Comment:
Maybe this function is useless, let the user directly call the `self.templates.TemplateResponse(...)` ?

Reply:
yeah I agree it will be simpler without this.